templates/cdap-ui-deployment.yaml

```diff
@@ -14,55 +14,52 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{.Name}}
-  namespace: {{.Master.Namespace}}
-{{if .Labels}}
+  name: {{.Base.Name}}
+  namespace: {{.Base.Namespace}}
+{{if .Base.Labels}}
   labels:
-    {{range $k,$v := .Labels}}
-    {{$k}}: '{{$v}}'
+    {{range $k,$v := .Base.Labels}}
+    {{$k}}: {{$v}}
```

Comment:
dont you need the single quotes ?

Reply:
Not sure. Some of our existing template use single quotes. Others don't. Seem to work without them, thus removing to make it consistent.

From quick search
https://stackoverflow.com/questions/19109912/do-i-need-quotes-for-strings-in-yaml
looks to be quotes are in generally not required